netket/models/autoreg.py

```diff
@@ -95,10 +104,73 @@ def conditionals(self, inputs: Array) -> Array:
           # it takes probability 0.3 to be spin down (local state index 0),
           # and probability 0.7 to be spin up (local state index 1).
         """
+        if inputs.ndim == 1:
+            inputs = jnp.expand_dims(inputs, axis=0)
```

Comment:
By the way, at some time we can remove such code to support inputs without the batch dimension in all models (Here we can see it's not covered by tests now)